docs/cli/graphql-transformer/directives.md

```diff
@@ -476,70 +478,126 @@ type Post
 }
 ```
 
-Owner authorization specifies that a user can access an object. To
-do so, each object has an *ownerField* (by default "owner") that stores ownership information
-and is verified in various ways during resolver execution.
+<amplify-callout>
+
+Owner authorization requires an authentication type of **Amazon Cognito User Pools** to be enabled in your app.
+
+</amplify-callout>
+
+Owner authorization specifies whether a user can access or operate against an object. To do so, each object will get an `ownerField` field (by default `owner` will be added to the object if not specified) that stores ownership information and is verified in various ways during resolver execution.
 
-You can use the *operations* argument to specify which operations are augmented as follows:
+You can use the `operations` argument to specify which operations are enabled as follows:
 
 - **read**: If the record's owner is not the same as the logged in user (via `$ctx.identity.username`), throw `$util.unauthorized()` in any resolver that returns an object of this type.
-- **create**: Inject the logged in user's `$ctx.identity.username` as the *ownerField* automatically.
-- **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
-- **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
+- **create**: Inject the logged in user's `$ctx.identity.username` as the `ownerField` automatically.
+- **update**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
+- **delete**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
 
-**Note**: When specifying operations as a part of the @auth rule, the operations not included in the list are not protected by default. For example, let's say you have the following schema:
+You must ensure that the `create` operations rule is either specified directly or indirectly to ensure that the owner's identity is stored with the record so it can be verified on subsequent requests.
```

Comment:
I would change "direct/indirect" to "is specified explicitly or inferred from defaults" and then say which is the case in the comments above the examples below. It's not clear why case #3 below is bad, and if it is bad why would you do it?

Reply:
Oh I was looking for a good way to word this and this nails it. Thank you!